### PR TITLE
fix error about deep option of watch in instance-methods.md

### PR DESCRIPTION
**Pull request by likev**

### CODE REVIEW

File: src/api/instance-methods.md [hunk 1]
```diff
@@ -121,7 +121,9 @@
 
 - **Option: deep**
 
-  To also detect nested value changes inside Objects, you need to pass in `deep: true` in the options argument. Note that you don't need to do so to listen for array mutations.
+  To also detect nested value changes inside Objects, you need to pass in `deep: true` in the options argument. Note that in Vue3 you **also** need to do so to listen for array mutations.
```
Comment (NataliaTepluhina) [inline src/api/instance-methods.md:124]: **suggestion**: this is a documentation for v3, I don't think we need an additional reminder about the current version here. WDYT about rephrasing:

```suggestion
  To also detect nested value changes inside Objects, you need to pass in `deep: true` in the options argument. This option also can be used to watch array mutations.
```

Comment (likev) [reply src/api/instance-methods.md:124]: agree with you

Comment (skirtles-code) [reply src/api/instance-methods.md:124]: @likev If you're happy with the change could you please merge it into your branch? You should see a button beneath the suggested change that will commit it for you. Thanks.

Comment (likev) [reply src/api/instance-methods.md]: There are a lot I need to learn about github, thanks!

**Review by NataliaTepluhina — APPROVED** [review]
@likev thank you for the implementing the suggestion!

**Pull request merged**